msg/JobFinished.msg

```diff
@@ -1,3 +1,3 @@
-string sender_id
-uint8 error_code
-string payload
\ No newline at end of file
+string sender_id # Name of the sending node
+uint8 error_code # '0' indicates success, everything else failure
```

Comment:
Are these (other than 0) error codes mapped somewhere ?

Reply:
They are the standard ERROR_SUCCESS and ERROR_FAILURE defines.